dev/dependencyList

```diff
@@ -132,23 +132,24 @@ metrics-core/4.2.8//metrics-core-4.2.8.jar
 metrics-jmx/4.2.8//metrics-jmx-4.2.8.jar
 metrics-json/4.2.8//metrics-json-4.2.8.jar
 metrics-jvm/4.2.8//metrics-jvm-4.2.8.jar
-netty-all/4.1.84.Final//netty-all-4.1.84.Final.jar
-netty-buffer/4.1.84.Final//netty-buffer-4.1.84.Final.jar
-netty-codec-dns/4.1.84.Final//netty-codec-dns-4.1.84.Final.jar
-netty-codec-http/4.1.84.Final//netty-codec-http-4.1.84.Final.jar
-netty-codec-http2/4.1.84.Final//netty-codec-http2-4.1.84.Final.jar
-netty-codec-socks/4.1.84.Final//netty-codec-socks-4.1.84.Final.jar
-netty-codec/4.1.84.Final//netty-codec-4.1.84.Final.jar
-netty-common/4.1.84.Final//netty-common-4.1.84.Final.jar
-netty-handler-proxy/4.1.84.Final//netty-handler-proxy-4.1.84.Final.jar
-netty-handler/4.1.84.Final//netty-handler-4.1.84.Final.jar
-netty-resolver-dns/4.1.84.Final//netty-resolver-dns-4.1.84.Final.jar
-netty-resolver/4.1.84.Final//netty-resolver-4.1.84.Final.jar
-netty-transport-classes-epoll/4.1.84.Final//netty-transport-classes-epoll-4.1.84.Final.jar
-netty-transport-native-epoll/4.1.84.Final/linux-aarch_64/netty-transport-native-epoll-4.1.84.Final-linux-aarch_64.jar
-netty-transport-native-epoll/4.1.84.Final/linux-x86_64/netty-transport-native-epoll-4.1.84.Final-linux-x86_64.jar
-netty-transport-native-unix-common/4.1.84.Final//netty-transport-native-unix-common-4.1.84.Final.jar
-netty-transport/4.1.84.Final//netty-transport-4.1.84.Final.jar
+netty-all/4.1.87.Final//netty-all-4.1.87.Final.jar
+netty-buffer/4.1.87.Final//netty-buffer-4.1.87.Final.jar
+netty-codec-dns/4.1.87.Final//netty-codec-dns-4.1.87.Final.jar
+netty-codec-http/4.1.87.Final//netty-codec-http-4.1.87.Final.jar
+netty-codec-http2/4.1.87.Final//netty-codec-http2-4.1.87.Final.jar
+netty-codec-socks/4.1.87.Final//netty-codec-socks-4.1.87.Final.jar
+netty-codec/4.1.87.Final//netty-codec-4.1.87.Final.jar
+netty-common/4.1.87.Final//netty-common-4.1.87.Final.jar
+netty-handler-proxy/4.1.87.Final//netty-handler-proxy-4.1.87.Final.jar
+netty-handler-ssl-ocsp/4.1.87.Final//netty-handler-ssl-ocsp-4.1.87.Final.jar
```

Comment:
it's a new jar, is it expected?

Reply:
It's mentioned in PR description. All I know is 1. `netty-handler-ssl-ocsp` package is released from 4.1.86+, and 2. it's included by `netty-all` which is a dependency of kyuubi server module.

Reply:
sgtm

Reply:
OK, addressed and updated the description for exclusion.